src/pages/saksbehandling/steg/beregning/Beregning.tsx

```diff
@@ -239,6 +240,10 @@ const Beregning = (props: VilkårsvurderingBaseProps) => {
                                     RemoteData.isSuccess(beregningStatus) ||
                                     (props.behandling.beregning && RemoteData.isInitial(beregningStatus))
                                 ) {
+                                    if (props.behandling.status === Behandlingsstatus.BEREGNET_AVSLAG) {
+                                        history.push(props.nesteUrl);
+                                    }
```

Comment:
Hvis simulering ikke skal startes i dette tilfellet må vi huske å returnere.
```suggestion
                                    if (props.behandling.status === Behandlingsstatus.BEREGNET_AVSLAG) {
                                        history.push(props.nesteUrl);
                                        return;
                                    }
```

Reply:
oops, ja!